frontend/app/templates/components/transportation/study-area-map.hbs

```diff
@@ -32,6 +32,13 @@
           data-test-popup='census-tract'
         />
       </Mapbox::LayerHoverCard>
+      <Mapbox::FeatureSelector
+        @map={{map}}
+        @layerId={{layer.layerId}} as |selectedFeatures| >
+        <Transportation::StudyAreaMap::CensusTractSelector
+          @selectedFeatureArray={{selectedFeatures}}
+          @analysis={{analysis}} />
+      </Mapbox::FeatureSelector>
```

Comment:
I'm a little confused by the difference between ::CensusTractSelector and ::FeatureSelector, because of naming. Seems like ::CensusTractSelector is responsible for mutating a specific array property on an object, and making sure it is unique. It doesn't really have anything to do with selections, no?

Reply:
Well "mutating a specific array property on an object" to my mind means selecting the census tract as part of your study selection, but alllllways happy to rename ! `studySelectionUpdater`?  `studySelectionHandler`? `studySelectionMutater`?